src/md/contributing.md

```diff
@@ -136,6 +136,8 @@ Creating a local branch that tracks a remote makes pull easier (no need to speci
   `git push origin +APEXCORE-123.my-feature`
   * Delete the extra squash branch:  
   `git branch -D APEXCORE-123.my-feature.squash`
+1. In case you need more time to work on the PR after receiving review feedback and anticipate some period of inactivity, please close the PR and let the reviewers know if you plan to work on the PR later. Once you have time to resume
```

Comment:
I think you can lead with the second sentence.

Reply:
@PramodSSImmaneni Please clarify or provide proposed version

Reply:
I meant you can lead with "When a PR is inactive longer than 2 months, it may be closed by a committer." then have the second sentence as "In case you need more time..."

Reply:
The primary goal is to ask contributors to manage PR and close/reopen PR. Note that it is also a contributor responsibility to find and convince a committer to merge his/her PR.

Reply:
It's fine. I just thought, since the closing of PR is the important thing there that will affect the contributor, it might be better to lead with that statement.

Reply:
Closing by a committer is a last resort action that preferably will not be needed if contributors follow the guidelines.